src/client/routes/VaultDetail/index.tsx

```diff
@@ -24,25 +24,25 @@ import {
 import { getConfig } from '@config';
 import { device } from '@themes/default';
 
-const BackButton = styled(Button)`
-  background-color: ${({ theme }) => theme.colors.surface};
-  color: ${({ theme }) => theme.colors.onSurfaceH2};
-`;
+// const BackButton = styled(Button)`
+//   background-color: ${({ theme }) => theme.colors.surface};
+//   color: ${({ theme }) => theme.colors.titles};
+// `;
 
-const StyledInfoCard = styled(InfoCard)`
+const StyledSliderCard = styled(SliderCard)`
   padding: 3rem;
   margin: auto;
 `;
 
-const ViewHeader = styled.div``;
+// const ViewHeader = styled.div``;
```

Comment:
Remove comments ?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6